springwolf-core/src/main/java/io/github/springwolf/core/asyncapi/scanners/common/payload/PayloadService.java

```diff
@@ -48,7 +48,9 @@ private NamedSchemaObject buildSchema(String contentType, Class<?> payloadType)
         String componentsSchemaName = this.componentsService.registerSchema(payloadType, contentType);
 
         SchemaObject schema = componentsService.resolveSchema(componentsSchemaName);
-        schema.setTitle(payloadType.getSimpleName());
+        if (schema != null) {
```

Comment:
this null check is mandatory -> bugfix

In the next step, the resolveSchema method should be removed.